tests/console/yast2_apparmor.pm

```diff
@@ -0,0 +1,143 @@
+# without any warranty.
+
+use strict;
+use base "consoletest";
+use testapi;
+
+
+
+sub run() {
+
+    select_console 'root-console';
+
+    # install yast2_apparmor package at first
+    assert_script_run("/usr/bin/zypper -n -q in yast2-apparmor");
+
+    # start apparmor configuration
+    script_run("/sbin/yast2 apparmor; echo yast2-apparmor-status-\$? > /dev/$serialdev", 0);
+
+    # check Apparmor Configuration is opened
+    assert_screen 'yast2_apparmor';
+    send_key 'ret';
+    assert_screen 'yast2_apparmor_enabled';
+
+    # part 1: open profile mode configuration and check toggle/show all profiles
+    send_key 'alt-n';
+    assert_screen 'yast2_apparmor_profile_mode_configuration';
+    send_key 'alt-o';
+    wait_still_screen;
+    assert_screen 'yast2_apparmor_profile_mode_configuration_show_all';
+    wait_still_screen;
+    send_key 'tab';
+    wait_still_screen;
+    send_key 'down';
+    wait_still_screen;
+    send_key 'alt-t';
+    assert_screen 'yast2_apparmor_profile_mode_configuration_toggle';
+    wait_still_screen;
+    send_key 'alt-b';
+    wait_still_screen;
+
+    # close apparmor configuration
+    send_key 'alt-d';
+    wait_still_screen;
+    assert_script_run("systemctl show -p ActiveState apparmor.service | grep ActiveState=active");
+
+    # part 2: start apparmor configuration again
+    script_run("/sbin/yast2 apparmor; echo yast2-apparmor-status-\$? > /dev/$serialdev", 0);
+    assert_screen 'yast2_apparmor';
+    send_key 'down';
+    assert_screen 'yast2_apparmor_configuration_manage_existing_profiles';
+    send_key 'ret';
+    assert_screen 'yast2_apparmor_configuration_manage_existing_profiles_edit';
+    send_key 'alt-p';
+    send_key 'down';
+
+    # add a profile
+    assert_screen 'yast2_apparmor_configuration_manage_existing_profiles_edit_add';
+    send_key 'alt-i';
+    wait_still_screen;
+    send_key 'alt-a';
+    wait_still_screen;
+    send_key 'alt-f';
+    assert_screen 'yast2_apparmor_configuration_manage_existing_profiles_edit_add_file';
+    send_key 'ret';
+    send_key 'alt-e';
+    type_string 'I_added_this_profile';
+    send_key 'alt-p';
+    wait_still_screen;
+    send_key 'down';
+    send_key 'up';
+    send_key 'spc';
+    send_key 'down';
+    send_key 'spc';
+
+    # confirm with cancel
+    send_key 'alt-c';
+    wait_still_screen;
+
+    # now add a directory with permission for read and write
+    send_key 'alt-a';
+    wait_still_screen;
+    send_key 'alt-d';
+    send_key 'alt-e';
+    type_string '/tmp';
+    sleep 2;
+    send_key 'alt-p';
+    wait_still_screen;
+    send_key 'down';
+    send_key 'up';
+    send_key 'spc';
+    send_key 'down';
+    send_key 'spc';
+    send_key 'alt-o';
+    wait_still_screen;
+    send_key 'alt-d';
+
+    # confirm to save changes to the profile
+    for (assert_screen 'yast2_apparmor_configuration_manage_existing_profiles_edit_file_changed') { send_key 'alt-y'; }
```

Comment:
this is just invalid syntax. for will just iterate over the object returned from assert_screen once. I.e. the for is useless.

```
coolo@nohostname#~>perl -e 'for (1) { print "Hallo\n"; }'
Hallo
```